ga4gh/backend.py

```diff
@@ -286,6 +292,18 @@ def startProfile(self):
     def endProfile(self):
         pass
 
+    def validateRequest(self, jsonDict, requestClass):
+        if ('REQUEST_VALIDATION' in flaskapp.app.config and
+                flaskapp.app.config['REQUEST_VALIDATION']):
+            if not requestClass.validate(jsonDict):
+                raise backendExceptions.RequestValidationFailureException
```

Comment:
Exceptions raised like this are implicitly instantiated.  Do we want to add parentheses as a matter of style?

Related: https://www.python.org/dev/peps/pep-0317/